modules/virt-vm-storage-volume-types.adoc

```diff
@@ -55,6 +54,14 @@ require persistent storage.
 Only RAW and QCOW2 formats are supported disk types for the container
 image registry. QCOW2 is recommended for reduced image size.
 
+[NOTE]
+====
+A containerDisk volume is ephemeral. It is discarded when
+the virtual machine is stopped, restarted, or deleted. A containerDisk
```

Comment:
Some people could say this is to greedy.
Under certain circumstances the image will remain, but the statement above is safe, thus even if greedy, let's keep it.